recipes/nodejs_from_source.rb

```diff
@@ -24,9 +24,12 @@
 
 case node['platform_family']
 when 'rhel', 'fedora', 'amazon'
-  package %w(openssl-devel python3 tar)
+  package %w(openssl-devel python2 tar)
```

Comment:
Why are you preferring python2 over python3?

Reply:
python3 doesn't work. Ran the test.

Reply:
ark and make are using python somewhere deep in the bowels to build from source.  python3 doesn't work.   Just installing python2 or python3 doesn't create a python link.  There are other options I suppose.

Reply:
I changed the install to use a temporary location for the python link.  The source install still installs python2 but there's no permanent link to point to it.  That's seems pretty minimal in terms of system changes.

Reply:
Yeah but that still seems like a sub-optimal way to do it. Let me ponder this and see if I can provide some feedback on a better way to do it. I don't like linking stuff in `/tmp` either as that will likely go away on reboots.

Reply:
I don't want to keep the link. It just needs to be there for the build. /tmp or a user specified location seems the most harmless place.  There's very little that is optimal about this cookbook :-). I need to get this one merged to use as a base for issues in the backlog.